eng/pipelines/vmr-build.yml

```diff
@@ -3,7 +3,7 @@
 # - installer-source-build (public)
 #   - PR: release/* and main, ultralite build
 # - installer-unified-build (public)
-#   - PR: release/* and main, lite build
+#   - PR: main, lite build
```

Comment:
Don't we need this to run on 9+ release branches?  We may want to leave release/* and add filters for 6,7, & 8

Reply:
I hope that this won't be the case and that we already work from dotnet/dotnet by that time.

Reply:
There will be release/9.0.1xx-preview branches though.  Don't we want this PR validation to run for them?

Reply:
Unified build isn't a shipping target so we shouldn't care if it's broken in a preview branch. IMO only main matter right now.

Reply:
That's a fair point for now.